Library/ViewModels/NoShippingPostCampaignCheckoutViewModelTests.swift

```diff
@@ -6,16 +6,22 @@ import ReactiveExtensions_TestHelpers
 import XCTest
 
 final class NoShippingPostCampaignCheckoutViewModelTests: TestCase {
-  private var vm = PostCampaignCheckoutViewModel(stripeIntentService: MockStripeIntentService())
+  private var vm = NoShippingPostCampaignCheckoutViewModel(stripeIntentService: MockStripeIntentService())
```

Comment:
I did check to make sure these tests passed before any of my other changes with the `NoShipping` version of the vm, and they did. If they hadn't, I would've created a separate pr to fix tests before adding new logic to the view model.